src/update/internal/updateactioncontroller.cpp

```diff
@@ -23,6 +23,7 @@
 
 #include "translation.h"
 #include "log.h"
+#include "globalmodule.h"
```

Comment:
there might be a better solution here?

Reply:
You can just remove `using namespace mu::framework;` instead, because nothing from `framework` is used.

Reply:
And it looks like `#include "translation.h"` and `#include "log.h"` can go too...

Reply:
That's indeed much better ;-)